go.mod

```diff
@@ -1,48 +1,50 @@
 module github.com/cybozu-go/moco-agent
 
-go 1.22.3
+go 1.22.5
 
 require (
 	github.com/cybozu-go/well v1.11.2
-	github.com/go-logr/logr v1.3.0
+	github.com/go-logr/logr v1.4.2
 	github.com/go-logr/stdr v1.2.2
 	github.com/go-logr/zapr v1.3.0
-	github.com/go-sql-driver/mysql v1.7.1
+	github.com/go-sql-driver/mysql v1.8.1
 	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
-	github.com/jmoiron/sqlx v1.3.5
-	github.com/onsi/ginkgo/v2 v2.13.2
-	github.com/onsi/gomega v1.30.0
-	github.com/prometheus/client_golang v1.17.0
+	github.com/jmoiron/sqlx v1.4.0
+	github.com/onsi/ginkgo/v2 v2.20.0
+	github.com/onsi/gomega v1.34.1
+	github.com/prometheus/client_golang v1.20.0
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/spf13/cobra v1.8.0
+	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
-	go.uber.org/zap v1.26.0
-	golang.org/x/sys v0.15.0
-	google.golang.org/grpc v1.59.0
-	google.golang.org/protobuf v1.31.0
+	go.uber.org/zap v1.27.0
+	golang.org/x/sys v0.24.0
+	google.golang.org/grpc v1.65.0
+	google.golang.org/protobuf v1.34.2
```

Comment:
Some packages seem to be still old (e.g. `github.com/onsi/ginkgo/v2`'s latest version is v2.20.2). 
`go get -u ./...` would update these.

Reply:
Thank you!
I think it's because this PR is created last month. 
I updated go.mod to latest now.